Mathlib/Data/Finset/Defs.lean

```diff
@@ -264,6 +264,9 @@ theorem Subset.antisymm {s₁ s₂ : Finset α} (H₁ : s₁ ⊆ s₂) (H₂ : s
 theorem subset_iff {s₁ s₂ : Finset α} : s₁ ⊆ s₂ ↔ ∀ ⦃x⦄, x ∈ s₁ → x ∈ s₂ :=
   Iff.rfl
 
+theorem not_mem_subset_iff : s ⊆ t ↔ ∀ ⦃a⦄, a ∉ t → a ∉ s := by
```

Comment:
I don't understand this name.

Reply:
Ehm coming back to this, me either...

Ah it is based on the name of the following lemma but had the implication the wrong way around.

```
Set.not_mem_subset {α} {a : α} {s t : Set α} (h : s ⊆ t) : a ∉ t → a ∉ s
```